alluxio-ui/master/src/containers/pages/Configuration/Configuration.tsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import {connect} from 'react-redux';
+import {Alert, Table} from 'reactstrap';
+import {Dispatch} from 'redux';
+
+import {IConfigTriple} from '../../../constants';
+import {IApplicationState} from '../../../store';
+import {fetchRequest} from '../../../store/config/actions';
+import {IConfig} from '../../../store/config/types';
+
+interface IPropsFromState {
+  errors: string;
+  loading: boolean;
+  config: IConfig;
+}
+
+interface IPropsFromDispatch {
+  fetchRequest: typeof fetchRequest;
+}
+
+interface IConfigurationProps {
+  refreshValue: boolean;
+}
+
+type AllProps = IPropsFromState & IPropsFromDispatch & IConfigurationProps;
+
+class Configuration extends React.Component<AllProps> {
+  public componentWillReceiveProps(props: AllProps) {
+    const {refreshValue} = this.props;
+    if (props.refreshValue !== refreshValue) {
+      this.props.fetchRequest();
+    }
+  }
+
+  public componentWillMount() {
+    this.props.fetchRequest();
+  }
+
+  public render() {
+    const {errors, config} = this.props;
+
+    if (errors) {
+      return (
+        <Alert color="danger">
+          Unable to reach the api endpoint for this page.
```

Comment:
Should we propagate these errors to the user, or is there somewhere we can find these errors later for debugging?